tools/niminst/buildsh.tmpl

```diff
@@ -123,7 +123,15 @@ case $ucpu in
   *power*|*ppc* )
     mycpu="powerpc" ;;
   *mips* )
-    mycpu="mips" ;;
+    mycpu="$("$CC" -dumpmachine | sed 's/-.*//')"
```

Comment:
And who says ``$CC`` supports ``-dumpmachine``? Why not distinguish via ``$upcu`` like the rest of the code?